release-notes/v2.4.0.md

```diff
@@ -0,0 +1,35 @@
+# HPE CSI Driver for Kubernetes 2.4.0 Release Notes
+
+## HPE CSI Driver for Kubernetes 2.4.0
+
+| **Version:** |2.4.0|
+|--------------|-----|
+| **Revision:** | Thursday September 21, 2023 |
+
+## Resolved Issues
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.4.0.
+
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+|CON-3002|csi.k8s|Updated all SIG storage sidecars to new supported versions|
+|CON-1744|csi.k8s|The default resource limits for NFS servers are set|
+|CON-1738|csi.k8s|Added anti-affinity rules to create a more even spread of NFS servers|
+|CON-417|csi.k8s|Support RWX access mode for raw block volumes in Nimble|
+|CON-2960|csi.k8s|The CSI driver will not make changes on the host when node conformance is disabled|
+|CON-3009|csi.k8s|ARM64 images support for the CSI driver|
+|CON-2568|csi.k8s|Enhancement of nfsNamespace parameter in Storage class|
+|CON-3015|csi.k8s|Fixed the funstionality of hpe-logcollector script|
+|CON-3025|csi.k8s|Added CRD files in CSI driver deployment through yaml files|
+
+## Known Issues
+
+The following table lists the known issues for HPE CSI Driver for Kubernetes v2.4.0. Please note that, `Known Issues` from previous releases are still applicable with suggested workarounds, if they are not part of `Resolved Issues` above.
+
+|ID|Component |Title|Description|Workaround|
```

Comment:
Why is there a workaround column without workaround content?

Reply:
We used the same template as before, we can remove it.

Reply:
Removed.